remappers/internal/metric.go

```diff
@@ -46,6 +46,12 @@ func AddMetrics(
 	ms.EnsureCapacity(ms.Len() + len(metrics))
 
 	for _, metric := range metrics {
+
+		//The translated metric should not be generated for a null or negative value
+		if (metric.IntValue != nil && *metric.IntValue <= 0) || (metric.DoubleValue != nil && *metric.DoubleValue <= 0) {
```

Comment:
```suggestion
		if (metric.IntValue != nil && *metric.IntValue <= 0) || (metric.DoubleValue != nil && *metric.DoubleValue <= 0 && math.IsInf(*metric.DoubleValue, 0) {
```
Should we also check for Infs here? Or is that expected to have?

Reply:
We can add the infinity check as an additional check.